src/components/Button.tsx

```diff
@@ -24,10 +24,9 @@ import styled, { css } from 'styled-components'
 import { sectionChildrenVariants } from './PageComponents/PageContainers'
 
 interface ButtonProps extends HTMLMotionProps<'button'> {
-  secondary?: boolean
-  alert?: boolean
+  mode?: 'primary' | 'secondary' | 'transparent'
```

Comment:
Any better idea for this prop name? `type` is taken by `HTMLMotionProps`...

Reply:
role?

Reply:
done!